core/src/main/java/org/aerogear/mobile/core/MobileCore.java

```diff
@@ -64,7 +66,10 @@ private MobileCore(final Context context, final Options options)
 
         // -- Parse JSON config file
         try (final InputStream configStream = context.getAssets().open(configFileName)) {
-            this.servicesConfig = MobileCoreJsonParser.parse(configStream);
+            MobileCoreJsonParser.parse(configStream);
```

Comment:
If you want to return the objects from `getXXX` methods, they should not be static variables: if you call the parse method 2 times, they will be overwritten.
So, if you want the `parse` method to be static, you should return an object with the result of the parsing.
If you don't want to return the results from the `parse` method but you want to return them with the `getXXX` methods, then make both the parse method, the instance variable and the `getXXX` methods not static.